### ISSUE TITLE
Add documentation at deployment_notes.md about Integromat, etc

### ISSUE THREAD

**Issue by ryancwalsh** [NEAR-Edu/near-certification-tools#35]

**Comment by petarvujovic98** [comment]
@ryancwalsh Two things that come to my mind immediately are:

1. Add the scenario schema to the repository, this might not be something we want since I believe that the schema generated from the scenario contains sensitive data (credentials) - if it doesn't than it's something to think about
2. Include a diagram, or some other type of visual, demonstrating the connection alongside a text document.

**Comment by ryancwalsh** [comment]
@petarvujovic98  Yeah we could check that the Scenario export doesn't contain email account credentials, our API key, etc. Or manually replace them with _____ after export. 

And some kind of text/visual explanation of how everything works will be helpful.

**Comment by ryancwalsh** [comment]
@petarvujovic98 Can you please include documentation for this (hopefully temporary) manual process (related to the calendar invite that I just sent you): https://airtable.com/appncY8IjPHkOVapz?automation=wflycUpvXCU1DtNHW
